package.json

```diff
@@ -143,8 +143,11 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "@concord-consortium/lara-interactive-api": "^1.0.3",
+    "@aws-sdk/client-s3": "^3.24.0",
+    "@aws-sdk/s3-request-presigner": "^3.24.0",
+    "@concord-consortium/lara-interactive-api": "^1.1.1",
```

Comment:
Should this be bumped to 1.1.2?

Reply:
Yes, once the new version of the library has been published.

Reply:
Done